source/blog/2016-01-14-atomic-app-0-3-1.html.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Atomic App 0.3.1 release. Persistent storage and more OpenShift support!
+author: cdrage
+date: 2016-01-14 17:00:00 UTC
+tags: atomic app, nulecule, composite apps, developers, alm, application lifecycle
+comments: true
+published: false
+---
+
+Today we've released [0.3.1](https://github.com/projectatomic/atomicapp/releases) of [Atomic App](http://www.projectatomic.io/docs/atomicapp/)!  This is our first release since the holiday break as well as the first release since 0.3.0. We've got quite a lot of new features added as well 
+
+The main features of this release are: 
+  * Native persistent storage integration. (Kubernetes only support for now)
+  * HTTPS (TLS) verification and support for OpenShift
+  * OpenShift `stop` added
+  * Nested Nulecule application support for OpenShift
+
+
+READMORE
+
+Bug fixes and changes:
+
+  * Refactor of our OpenShift provider
+  * Compatibility fixes with Atomic CLI and Atomic App integration
+  * CLI restructure to allow params at all levels of interaction
```

Comment:
How about his instead: `Addition of several CLI parameters to allow for passing provider options on the command line`